v2/docs/ProvidersApi.md

```diff
@@ -0,0 +1,96 @@
+# \ProvidersApi
+
+All URIs are relative to *https://console.jumpcloud.com/api/v2*
+
+Method | HTTP request | Description
+------------- | ------------- | -------------
+[**ProvidersListAdministrators**](ProvidersApi.md#ProvidersListAdministrators) | **Get** /providers/{provider_id}/administrators | List Provider Administrators
+[**ProvidersPostAdmins**](ProvidersApi.md#ProvidersPostAdmins) | **Post** /providers/{provider_id}/administrators | Create a new Provider Administrator
+
+
+# **ProvidersListAdministrators**
+> InlineResponse200 ProvidersListAdministrators(ctx, providerId, contentType, accept, optional)
+List Provider Administrators
+
+This endpoint returns a list of the Administrators associated with the Provider.  #### Sample Request ``` curl -X GET https://console.jumpcloud.com/api/v2/providers/{ProviderID}/administrators \\   -H 'Accept: application/json' \\   -H 'Content-Type: application/json' \\   -H 'x-api-key: {API_KEY}' ```
```

Comment:
Should the `#### Sample...` start on a new line?

Reply:
That's also just how it's automatically generated.

Reply:
The reason I asked was it seemed like it was on a newline in other places but just must be how the spacing worked out.